src/Polly.Shared/PolicyResult.cs

```diff
@@ -103,12 +116,25 @@ internal PolicyResult(TResult result, OutcomeType outcome, Exception finalExcept
         /// </summary>
         public Context Context { get; }
 
-        internal static PolicyResult<TResult> Successful(TResult result, Context context)
+        /// <summary>
+        /// Builds a successful <see cref="PolicyResult" />
```

Comment:
((suggest change to))

`Builds a <see cref="PolicyResult" /> representing a successful execution through the policy.`